lib/util/platform.js

```diff
@@ -617,21 +617,26 @@ shaka.util.Platform = class {
       const hasCanDisplayType = window.cast && cast.__platform__ &&
           cast.__platform__.canDisplayType;
 
-      // Most Chromecasts can do 1080p.
-      maxResolution.width = 1920;
-      maxResolution.height = 1080;
+      // Some hub devices can only do 720p.  Default to that.
+      maxResolution.width = 1280;
+      maxResolution.height = 720;
 
       try {
         if (hasCanDisplayType && await cast.__platform__.canDisplayType(
```

Comment:
nit; Up to your preference, just in case you missed it - you might check the presence of `hasCanDisplayType` just once.

Reply:
I tried it, but I think the extra layer of `if` outside the try/catch makes the method overall harder to read and follow.  The runtime difference is negligible, so I'm leaving.